src/gcp/storage.ts

```diff
@@ -214,14 +213,12 @@ export function deleteObject(location: string): Promise<any> {
  * Gets a storage bucket from GCP.
  * Ref: https://cloud.google.com/storage/docs/json_api/v1/buckets/get
  * @param {string} bucketName name of the storage bucket
- * @returns a bucket resource object
+ * @return a bucket resource object
  */
 export async function getBucket(bucketName: string): Promise<BucketResponse> {
   try {
-    const result = await api.request("GET", `/storage/v1/b/${bucketName}`, {
-      auth: true,
-      origin: api.storageOrigin,
-    });
+    const localAPIClient = new Client({ urlPrefix: storageOrigin });
```

Comment:
Oh! I remember this now. Because of the way the URL is formatted.

Since the version number isn't the _first_ part of the path, I had to create a different API client that didn't add the default version in. The other option would be to extend the `urlPrefix` with `storage` and keep `v1`, but that seemed like an equal amount of head-scratching for the reader.